blazor/speeddial/appearance.md

```diff
@@ -0,0 +1,128 @@
+---
+layout: post
+title: Appearance in Blazor SpeedDial Component | Syncfusion
+description: Checkout and learn here all about appearance and how to use them in Syncfusion SpeedDial component and much more.
+platform: Blazor
+control: SpeedDial
+documentation: ug
+---
+
+# Appearance in Blazor SpeedDial Component
+
+This section briefly explains about Appearance of Speed dial.
+
+## Disabled
+
+You can enable or disable the Speed Dial Component using [Disabled](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Buttons.SfSpeedDial.html#Syncfusion_Blazor_Buttons_SfSpeedDial_Disabled) property.
+
+{% tabs %}
+{% highlight razor %}
+
+@using Syncfusion.Blazor.Buttons
+
+<SfSpeedDial Disabled=true Content="Edit">
+    <SpeedDialItems>
+        <SpeedDialItem IconCss="e-icons e-cut"/>
+        <SpeedDialItem IconCss="e-icons e-copy"/>
+        <SpeedDialItem IconCss="e-icons e-paste"/>
+    </SpeedDialItems>
+</SfSpeedDial>
+
+{% endhighlight %}
+{% endtabs %}
+
+![Blazor SpeedDial component with disabled](./images/Blazor-SpeedDial-Disabled.png)
+
+## Visibility
+
+You can set the Speed Dial Component to visible/hidden state using [Visible](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Buttons.SfSpeedDial.html#Syncfusion_Blazor_Buttons_SfSpeedDial_Visible) property.
+
+{% tabs %}
+{% highlight razor %}
+
+@using Syncfusion.Blazor.Buttons
+
+<SfSpeedDial Visible=false Content="Edit">
+    <SpeedDialItems>
+        <SpeedDialItem IconCss="e-icons e-cut"/>
+        <SpeedDialItem IconCss="e-icons e-copy"/>
+        <SpeedDialItem IconCss="e-icons e-paste"/>
+    </SpeedDialItems>
+</SfSpeedDial>
+
+{% endhighlight %}
+{% endtabs %}
+
+## Modal
+
+You can use the [IsModal](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Buttons.SfSpeedDial.html#Syncfusion_Blazor_Buttons_SfSpeedDial_IsModal) property to set the Speed Dial as modal which adds an overlay to prevent the background interaction.
```

Comment:
You can use the [IsModal]() property to open the Speed Dial items as modal which adds an overlay to prevent the background interaction.

Reply:
Modified the content as given